service/host.go

```diff
@@ -164,6 +164,7 @@ func (s *manager) shutdown(err error, reason string, exitCode *int) (bool, error
 	}
 
 	// Flush tracing buffers
+	// GFM-378 uhttp.Client is not safe to send requests anymore
```

Comment:
If clients get dispatcher with yarpc.Dispatcher(), but don't initialize the module, then it will be nil and users will not be able to create client..